packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
The installation section and basic use should come right at the top. See https://loopback.io/doc/en/contrib/README-guidelines.html